examples/kafka_message_consumer_group_service/kafka_message_consumer_group_service.description

```diff
@@ -0,0 +1,2 @@
+// Ballerina natively supports Kafka. Here is an example in which we create
```

Comment:
Not sure if "natively supports" is the correct phrase to use here. It's just a library